create_bot_knowledge.py

```diff
@@ -21,20 +21,22 @@
 import os
 import json
 
-from typing import List
-from typing import Tuple
+from typing import List, Tuple, Dict, Optional, Union, Set
 from pathlib import Path
 
-from github import Github
+
 from ogr.services.github import GithubService
-from ogr.abstract import Issue, IssueComment, IssueStatus, PullRequest, PRComment, PRStatus, CommitComment
+from ogr.abstract import PRComment, PRStatus, CommitComment, GitProject
+
+from github import Github, GithubObject, Issue, IssueComment, PullRequest, PullRequestReview
+from github.Repository import Repository
 
 _LOGGER = logging.getLogger(__name__)
 logging.basicConfig(level=logging.INFO)
 
 _GITHUB_ACCESS_TOKEN = os.getenv("GITHUB_ACCESS_TOKEN")
 
-PROJECTS = [
+PROJECTS = os.getenv("PROJECTS") #[
```

Comment:
I like the idea, but I don't see where you actually use `PROJECTS`

Reply:
i've changed main to the function which takes projects as parameter, will remove this env variable